roles/create-openshift-resources/tasks/create_app.yml

```diff
@@ -15,21 +20,23 @@
   set_fact:
     app_options: ''
     app_source: ''
+    labels: ''
+    environment_variables: ''
 
 - name: "Set App Source with Git Url and Base Image"
   set_fact:
-    app_source: "{{ app.base_image }}~{{ app.git_url }}"
-  when: app.git_url is defined and app.git_url != '' and app.base_image is defined and app.base_image != ''
+    app_source: "{{ app.base_image }}~{{ app.scm_url }}"
+  when: app.scm_url is defined and app.scm_url != '' and app.base_image is defined and app.base_image != ''
 
 - name: "Set App Source with only Base Image"
   set_fact:
     app_options: "{{ app_options }} --image-stream={{ app.base_image }}"
-  when: (app.base_image is defined and app.base_image != '') and (app.git_url is not defined or app.git_url == '')
+  when: (app.base_image is defined and app.base_image != '') and (app.scm_url is not defined or app.scm_url == '')
 
 - name: "Set App Source with only Git Url"
```

Comment:
@sherl0cks no - my comment is for line 29/36 - i.e.: the "- name: "Set App Source with only Git Url".